qcelemental/molparse/nucleus.py

```diff
@@ -248,17 +249,17 @@ def offer_user_label(lbl):
 
     text = ['', """--> Inp: A={}, Z={}, E={}, mass={}, real={}, label={}""".format(A, Z, E, mass, real, label)]
 
-    Z_exact = []  # *_exact are candidates for the final value
-    Z_range = []  # *_range are tests that the final value must pass to be valid
-    A_exact = []
-    A_range = []
-    m_exact = []
-    m_range = []
+    Z_exact: List = []  # *_exact are candidates for the final value
```

Comment:
Any reason to type these inside functions?

Reply:
only that mypy raised alerts and I knew how to fix them

Reply:
Gotcha.